### PR TITLE
UCAS API Changes

### PR DESCRIPTION
**Pull request by Skumarhpa**

### CODE REVIEW

File: src/Sfa.Tl.ResultsAndCertification.Application/Services/UcasDataAbbreviations.cs [hunk 1]
```diff
@@ -55,6 +73,12 @@ public static string GetAbbreviatedResult(UcasResultType ucasResultType, string
                 case UcasResultType.OverallResult:
                     hasValue = _overallResultsAbbreviations.TryGetValue(result, out abbreviatedResult);
                     break;
+                case UcasResultType.IndustryPlacement:
+                    hasValue = _industryPlacementResultsAbbreviations.TryGetValue(result,out abbreviatedResult);
+                    break;
+                //case UcasResultType.DualSpecialsms:
```
Comment (mustafajawad786) [inline src/Sfa.Tl.ResultsAndCertification.Application/Services/UcasDataAbbreviations.cs:79]: Please remove this commented code.

File: src/Tests/Sfa.Tl.ResultsAndCertification.Functions.UnitTests/Services/UcasDataTransferService/ProcessUcasDataRecords/When_UcasDualDataRecords_AreNotFound.cs [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                   }
+
+                  } }
+            };
+
+            return pathWayData;
+      
+    }
+
+
```
Comment (mustafajawad786) [inline src/Tests/Sfa.Tl.ResultsAndCertification.Functions.UnitTests/Services/UcasDataTransferService/ProcessUcasDataRecords/When_UcasDualDataRecords_AreNotFound.cs:80]: Please remove these unnecessary lines.

File: src/Tests/Sfa.Tl.ResultsAndCertification.Functions.UnitTests/Services/UcasDataTransferService/ProcessUcasDataRecords/When_UcasDualDataRecords_AreFound.cs [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+                   }
+
+                  }
+                }
+            };
+
+            return pathWayData;
+
+        }
+
```
Comment (mustafajawad786) [inline src/Tests/Sfa.Tl.ResultsAndCertification.Functions.UnitTests/Services/UcasDataTransferService/ProcessUcasDataRecords/When_UcasDualDataRecords_AreFound.cs:115]: Please remove these unnecessary lines.

Comment (Skumarhpa) [reply src/Tests/Sfa.Tl.ResultsAndCertification.Functions.UnitTests/Services/UcasDataTransferService/ProcessUcasDataRecords/When_UcasDualDataRecords_AreFound.cs]: removed

**Review by mustafajawad786 — APPROVED** [review]

**Pull request merged**